.gitignore

```diff
@@ -133,3 +133,4 @@ helm/defectdojo/charts
 docs/public
 docs/node_modules
 docs/resources
+.pre-commit-config.yaml
```

Comment:
This additional line in .gitignore seems not to be part of the bugfix. Please remove it.

Reply:
sure